src/quo2/components/drawers/bottom_actions/style.cljs

```diff
@@ -0,0 +1,26 @@
+(ns quo2.components.drawers.bottom-actions.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(def buttons-container
+  {:flex-direction     :row
+   :justify-content    :space-around
+   :padding-horizontal 20
+   :padding-vertical   12})
+
+(def button-container
+  {:flex 1})
+
+(def button-container-2-actions
+  (merge button-container {:margin-right 12}))
+
+(def description
+  {:color              colors/neutral-50
+   :text-align         :center
+   :padding-horizontal 40})
+
+(def scroll
```

Comment:
did you check with designers to see what the difference is when `scroll` is enable or not? it's not so clear to see from the designs so I'm not sure what to check here.